midi_app_controller/gui.py

```diff
@@ -0,0 +1,426 @@
+import sys
```

Comment:
Please adjust order of import statements: standard library, external libraries, local files. Groups separated by new lines. This convention is used in napari code.